webapps/docs/changelog.xml

```diff
@@ -47,6 +47,10 @@
 <section name="Tomcat 9.0.23 (markt)" rtext="in development">
   <subsection name="Catalina">
     <changelog>
+      <fix>
+        <bug>63636</bug>: <code>Context.findRoleMapping()</code> never called
+        in <code>RealmBase#hasRole()</code>. (michaelo)
+      </fix>
       <update>
```

Comment:
Please see the notes at the top of this file regarding the ordering of sections and the ordering of elements in each section.

Reply:
Pardo, I did not notice this. Will change, no issue.

Reply:
BZ issues should appear in order so the entry for BZ 63636 should appear after BZ 63608.

Reply:
Done.